ocrspace/main.py

```diff
@@ -1,4 +1,5 @@
 import requests
+from enum import (Enum, unique)
```

Comment:
Remove the unique import and the parentheses

Reply:
resolved on c9ffd78ddc93ede49af145fe706ee269c4e0a42c